app/serializers/tag_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class TagSerializer < ActiveModel::Serializer
+
+ attributes :name
+ has_many :notes, through: :taggings
```

Comment:
Don't need the `through:` here. `has_many` in a serializer doesn't mean the same thing as `has_many` in a model. Here, it mostly means "There are several of these, so make it an array"